components/Modals/NewPostModal.tsx

```diff
@@ -18,33 +18,111 @@ import {
 } from "react";
 import { StyleSheet, css } from "aphrodite";
 import { useRouter } from "next/router";
+import {
+  DEFAULT_POST_BUTTON_VALUES,
+  NewPostButtonContext,
+  NewPostButtonContextType,
+} from "~/components/contexts/NewPostButtonContext";
+import { getIsOnMobileScreenSize } from "~/config/utils/getIsOnMobileScreenSize";
 import BaseModal from "./BaseModal";
 import Button from "../Form/Button";
 import Link from "next/link";
 import Modal from "react-modal";
 import PaperUploadWizardContainer from "../Paper/UploadWizard/PaperUploadWizardContainer";
 import ResearchhubOptionCard from "../ResearchhubOptionCard";
-import {
-  DEFAULT_POST_BUTTON_VALUES,
-  NewPostButtonContext,
-  NewPostButtonContextType,
-} from "~/components/contexts/NewPostButtonContext";
+import killswitch from "~/config/killswitch/killswitch";
 
 export type NewPostModalProps = {
   currentUser: any;
 };
 
+export const getModalOptionItems = (currentUser) => [
+  {
+    key: "paper_upload",
+    header: "Upload a Paper",
+    description:
+      "Upload a paper that has already been published. Upload it via a link to the journal, or upload the PDF directly.",
+    icon: (
+      <PaperIcon
+        height={40}
+        onClick={silentEmptyFnc}
+        width={40}
+        withAnimation={false}
+      />
+    ),
+  },
+  {
+    key: "eln",
+    header: "Publish a Post",
+    description:
+      "All posts must be academic in nature. Ideas, theories, and questions to the community are all welcome.",
+    onClick: async () => {
+      /* @ts-ignore */
+      const note = await createNewNote({
+        orgSlug: currentUser.organization_slug,
+        grouping: NOTE_GROUPS.WORKSPACE,
+      });
+      /* @ts-ignore */
+      router.push(`/${currentUser.organization_slug}/notebook/${note.id}`);
+    },
+    icon: (
+      <PostIcon
+        height={40}
+        onClick={silentEmptyFnc}
+        width={40}
+        withAnimation={false}
+      />
+    ),
+  },
+  {
+    key: "hypothesis",
+    header: "Create a Meta-Study",
+    description:
+      "Aggregate a collection of papers that support a particular scientific theory",
+    route: "/hypothesis/create",
+    icon: (
+      <HypothesisIcon
+        height={40}
+        onClick={silentEmptyFnc}
+        width={40}
+        withAnimation={false}
+      />
+    ),
+  },
+  killswitch("bountyQuestion")
+    ? {
+        key: "question",
+        header: "Ask a Question",
+        description:
+          "All posts must be scientific in nature. Ideas, theories, and questions to the community are all welcome.",
```

Comment:
We will need to update the text here before go live but it's fine atm

Reply:
do we have new text you want to use @yattias ?